ci-bootstrap/README.md

```diff
@@ -0,0 +1,9 @@
+# Resource for setting up role in a new AWS account.
+
+# When setting up a new infrastructure for the lambda, first run deploy.sh script to create a correct role with correctly assigned policies.
+# Replace ENVIRONMENT variable <accountname> with sandbox, cicd, staging or prod
```

Comment:
Good to see extra info added :+1: and can you look at the formatting? The README.md files are in [markdown](https://daringfireball.net/projects/markdown/syntax). Take a quick look at that link if you've not used it before (it's the same as is used in slack - although slack only uses a subset). Always happy to do a run through :)

If you want to know how the file will look when it's rendered, you can hit the little document button on the top right of this diff view (between the *<>* and the Viewed buttons) and it'll show you a preview.

<img width="208" alt="Screenshot 2020-05-20 at 09 03 11" src="https://user-images.githubusercontent.com/1521281/82421644-32aadc00-9a79-11ea-9248-50a6778f7c4d.png">

Reply:
And VS Code has a preview option which really helps to visualise-as-you-type.

Reply:
Thanks @bambrp and @necrophonic - done